### PR TITLE
CI: Fix build error on Windows

### PR DESCRIPTION
**Pull request by ashie**
Try to fix the following error:

```
Failed to download gem files: <bundle package --no-install> (stdout: Fetching gem metadata from https://rubygems.org/...........
Fetching https://github.com/fluent/fluentd
, stderr: Warning: the running version of Bundler (2.2.9) is older than the version that created the lockfile (2.2.32). We suggest you to upgrade to the version that created the lockfile by running `gem install bundler:2.2.32`.
nokogiri-1.13.1-x64-mingw32 requires ruby version < 3.1.dev, >= 2.6, which is
incompatible with the current version, ruby 2.5.9p229
```

Signed-off-by: Takuro Ashie <ashie@clear-code.com>

### CODE REVIEW

**Comment by ashie** [comment]
Hmm, another error is occurred. What's this?
```
git archive HEAD --prefix td-agent-4.3.0/ --output D:/a/fluent-package-builder/fluent-package-builder/td-agent/td-agent-4.3.0.tar.gz
tar xvf D:/a/fluent-package-builder/fluent-package-builder/td-agent/td-agent-4.3.0.tar.gz
tar: Cannot connect to D: resolve failed
rake aborted!
Command failed with status (128): [tar xvf D:/a/fluent-package-builder/fluent...]
D:/a/fluent-package-builder/fluent-package-builder/td-agent/Rakefile:1205:in `block in build_archive'
D:/a/fluent-package-builder/fluent-package-builder/td-agent/Rakefile:1201:in `build_archive'
D:/a/fluent-package-builder/fluent-package-builder/td-agent/Rakefile:1108:in `block in define_archive_task'
D:/a/fluent-package-builder/fluent-package-builder/td-agent/Rakefile:1481:in `block (2 levels) in define'
Tasks: TOP => msi:dockerbuild => td-agent-4.3.0.tar.gz
```

**Comment by ashie** [comment]
Building cmetrics failed:
https://github.com/fluent/fluent-package-builder/runs/4897918349?check_suite_focus=true
```
2022-01-21T15:15:40.3940937Z Running 'configure' for cmetrics 0.2.2... *** extconf.rb failed ***
2022-01-21T15:15:40.3941586Z Could not create Makefile due to some reason, probably lack of necessary
2022-01-21T15:15:40.3942013Z libraries and/or headers.  Check the mkmf.log file for more details.  You may 
2022-01-21T15:15:40.3942395Z need configuration options.
2022-01-21T15:15:40.3942632Z 
2022-01-21T15:15:40.3942827Z 
2022-01-21T15:15:40.3943086Z Provided configuration options:
2022-01-21T15:15:40.3943407Z         --with-opt-dir
2022-01-21T15:15:40.3943717Z         --without-opt-dir
2022-01-21T15:15:40.3944017Z         --with-opt-include
2022-01-21T15:15:40.3944470Z         --without-opt-include=${opt-dir}/include
2022-01-21T15:15:40.3944948Z         --with-opt-lib
2022-01-21T15:15:40.3945274Z         --without-opt-lib=${opt-dir}/lib
2022-01-21T15:15:40.3945610Z         --with-make-prog
2022-01-21T15:15:40.3945908Z         --without-make-prog
2022-01-21T15:15:40.3946199Z         --srcdir=.
2022-01-21T15:15:40.3946442Z         --curdir
2022-01-21T15:15:40.3947088Z         --ruby=C:/opt/td-agent/bin/$(RUBY_BASE_NAME)
2022-01-21T15:15:40.3947482Z C:/opt/td-agent/lib/ruby/gems/2.7.0/gems/mini_portile2-2.7.1/lib/mini_portile2/m
2022-01-21T15:15:40.3948011Z ini_portile.rb:429:in
2022-01-21T15:15:40.3948361Z `spawn': Permission denied - cmake (Errno::EACCES)
2022-01-21T15:15:40.3948760Z from
2022-01-21T15:15:40.3949025Z C:/opt/td-agent/lib/ruby/gems/2.7.0/gems/mini_portile2-2.7.1/lib/mini_portile2/m
2022-01-21T15:15:40.3949497Z ini_portile.rb:429:in
2022-01-21T15:15:40.3949727Z `block in execute'
2022-01-21T15:15:40.3950055Z from
2022-01-21T15:15:40.3950327Z C:/opt/td-agent/lib/ruby/gems/2.7.0/gems/mini_portile2-2.7.1/lib/mini_portile2/m
```

@cosmo0920 Could you resolve this?

**Comment by cosmo0920** [comment]
Hmm..., Let me check it.

**Comment by cosmo0920** [comment]
I couldn't get succeeded to start Dockerized Windows package building environment.

**Comment by cosmo0920** [comment]
We got a green for Windows CI task!!!!
Could you review my changes, @ashie?

**Comment by ashie** [comment]
Thanks!

File: td-agent/Rakefile [hunk 1]
```diff
@@ -160,6 +160,14 @@ def render_template(dest, src, config, opts={})
   end
 end
 
+def tar_command
+  if windows?
+    ["ridk", "exec", File.join(RubyInstaller::Runtime.msys2_installation.msys_path, "usr", "bin", "tar.exe")]
```
Comment (ashie) [inline td-agent/Rakefile:165]: Probably `["ridk", "exec", "tar"]` is enough for it.

Comment (cosmo0920) [reply td-agent/Rakefile:165]: Ah, yes. I'll fix it.

Comment (cosmo0920) [reply td-agent/Rakefile]: https://github.com/fluent/fluent-package-builder/pull/364/commits/942834e52d84ae0cedad0b2150a7620f9614cc92 probably will reduce a redundant line.

**Comment by ashie** [comment]
Thanks! I'll merge this after passing CI.

**Comment by cosmo0920** [comment]
🚀

**Pull request merged**